casper/src/main/scala/coop/rchain/casper/api/BlockApi.scala

```diff
@@ -82,6 +85,8 @@ object BlockApi {
   type Error     = String
   type ApiErr[A] = Either[Error, A]
 
+  final class BlockApiException(message: String) extends Exception(message)
+
```

Comment:
This definition already presented in WebApi.scala but it makes more sense to place it here.